docs/snapcraft-parts.md

```diff
@@ -164,3 +164,16 @@ If you only need to embed a Java runtime, add a part with the jdk type. This
 will pull a relocatable OpenJDK via the default-jdk Ubuntu package and will
 set the proper `JAVA_HOME` and `PATH` environment variables in wrapper
 scripts generated by snapcraft or when running the app locally.
+
+
+## Reusing parts
+
+Successful snapcraft parts can be published in
+[the wiki][wiki] and reused by other snapcraft.yaml files.  A part that does
+not include the 'plugin' option will be searched for on [the wiki][wiki] and
+the content from there will be used to build the part locally.  As the number
+of published parts increases, writing a snapcraft.yaml file will get easier
+because one would simply include a set of published parts and the local part(s)
+specific to the snap being created.
+
+[wiki]: https://wiki.ubuntu.com/Snappy/Parts
```

Comment:
Updated.  I added an explanation of the wiki entry format and an example.  I also added a note saying it can take up to an hour for the wiki update to become available to snapcraft.